src/esphomelib/time/sntp_component.h

```diff
@@ -0,0 +1,52 @@
+//
+//  sntp_component.h
+//  esphomelib
+//
+//  Created by Brad Davidson on 06.06.18.
+//  Copyright © 2018 Otto Winter. All rights reserved.
+//
+
+#ifndef ESPHOMELIB_SNTP_COMPONENT_H
+#define ESPHOMELIB_SNTP_COMPONENT_H
+
+
+#include "esphomelib/component.h"
+#include "esphomelib/defines.h"
+
+#ifdef ARDUINO_ARCH_ESP32
+  #include "apps/sntp/sntp.h"
+#endif
+#ifdef ARDUINO_ARCH_ESP8266
+  #include "sntp.h"
+#endif
```

Comment:
If it's not used in the header file, this include should go into the `.cpp` file.